sql/sql_bitmap.h

```diff
@@ -133,8 +133,12 @@ template <uint width> class Bitmap
   }
   bool is_prefix(uint prefix_size) const
   {
-    DBUG_ASSERT(prefix_size <= width);
-
+    // if prefix_size is larger than width, the function would return false
```

Comment:
As @vaintroub  said on the MDEV, this is the wrong place to fix it. It will probably have consequences for other uses of the Bitmap.

The solution is in `generate_derived_keys_for_table` or earlier to prevent a prefix_size of 65 being passed here.